lib/miq_expression/query_helper.rb

```diff
@@ -0,0 +1,107 @@
+class MiqExpression
+  class QueryHelper < Arel::Visitors::Visitor
+    # rubocop:disable Naming/MethodName (the visitor pattern is defined with odd naming conventions)
+    if ActiveRecord.version.to_s < "5.2"
```

Comment:
I want to merge into master (5.1) and not rely upon having to be running 5.2

Reply:
Why?  Do you want to backport this?

Unless we find a reason to backport, let's just ~~git~~ get the MVP code necessary that is needed for 5.2 only instead of dragging a bunch of code along that will never get used in production anyway.

Reply:
done - removed